### ISSUE TITLE
Other Lighthouse Metrics

### ISSUE THREAD

**Issue by mgifford** [cloudfour/lighthouse-parade#106]
I assume it would be pretty trivial to collect other metrics, for instance accessibility, via Lighthouse Parade.

Could that be something that is just enabled optionally from the command line? 

Really interesting to explore sites this way. Thanks.

**Comment by calebeby** [comment]
Hi @mgifford! Lighthouse-parade is focused on Lighthouse's quantitative metrics.

Are there specific quantitative metrics in the Accessibility section of Lighthouse (other than the overall "score") that would be helpful to add? From what I've seen, the details that Lighthouse lists under Accessibility are usually suggestions for improvements, but without corresponding quantitative metrics to measure by.

**Comment by mgifford** [comment]
I'm not sure on this. I know that for accessibility it leverages Deque's axe for the accessibility engine.  There are specific elements in axe that could be pulled out, but not sure if they are exposed in Lighthouse at all. Sorry.

**Comment by calebeby** [comment]
@mgifford We are thinking about the best way to implement this, but yes, this is definitely a feature we want! I'll update this issue once I get something running!

**Comment by Adilius** [comment]
Just adding my opinion here, adding columns for accessibility score, number of faults, perhaps which WCGA rule failed etc. would really help my work
